.eslintrc

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "vtex-react/gatsby",
   "root": true,
+  "parserOptions": {
+    "ecmaVersion": 2020
```

Comment:
The previous value was 2019. Changing to at least 2020 fixed this:
<img width="697" alt="CleanShot 2022-09-21 at 17 13 36@2x" src="https://user-images.githubusercontent.com/381395/191601433-df0b8a0b-aa82-446f-af28-be7ec6ba3ddf.png">